department-of-veterans-affairs/caseflow
# Motions to Vacate | Set up demo data in seeds

### Description
This PR sets up appeals for working with Motions to Vacate at various stages throughout the process, for use during dogfooding, UAT, and training.

### Testing Plan
1. Run `make build` to re-do seeds
1. Check out the new MTV data in the development environment
